core/src/uix/compiler/aot.clj

```diff
@@ -446,6 +446,7 @@
 
 (defmethod compile-element :component [v]
   (let [[tag & args] v
+        tag (vary-meta tag assoc :tag 'js)
```

Comment:
It doesn't make much difference here, but I'd use [`with-meta`](https://clojuredocs.org/clojure.core/with-meta), (`vary-meta` is using `with-meta` internally)

```clj
tag (with-meta tag {:tag 'js})
```

Reply:
I assumed that `with-meta` would replace it, and `vary-meta` would append it, so that's why I used it. I don't know if this is true, tho, or if it actually matters here.

Reply:
Yes, `vary-meta` would append and preserve current meta. That's actually makes more sense to use it then 👍